integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -112,6 +114,26 @@ class RegionForm(CustomModelForm):
         ),
     )
 
+    duplication_pbo_behavior = forms.ChoiceField(
+        choices=(
```

Comment:
The choices could go into the constants directory, to be more consistent with the rest of the code (and prevent typos)

Reply:
Right, good point.